src/include/settings/settings_defs.h

```diff
@@ -287,6 +310,14 @@ SETTING_bool(
     noisepage::settings::Callbacks::MetricsQueryTrace
 )
 
+SETTING_string(
+    query_trace_metrics_output,
+    "Output type for Query Traces Metrics (default: CSV)",
```

Comment:
Maybe document what are the valid options.

Reply:
Documented.